packages/playwright/src/matchers/expect.ts

```diff
@@ -265,16 +265,6 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
         const message = jestError.message;
         if (customMessage) {
           const messageLines = message.split('\n');
-          // Jest adds something like the following error to all errors:
```

Comment:
Why remove this code?

Reply:
This code was doing something weird in a weird place in a wrong way. Tell me what it tried to do and maybe we find a better place.